src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,13 @@
+package seedu.addressbook.data.person;
+
+/** 
+ * 
+ * Interface for printables: Name, Phone, Email, and Address
+ *
+ */
+
```

Comment:
avoid a line of space between header comments and the class/method/interface. Also avoid mentioning what classes implement this interface.